.env.example

```diff
@@ -13,3 +13,5 @@ REACTION_USER_NAME=admin
 REACTION_USER=Admin
 ROOT_URL=http://localhost:3000
 IDENTITY_PROVIDER_MODE=all
+GRAPHQL_INTROSPECTION_ENABLED=true
+GRAPHQL_PLAYGROUND_ENABLED=true
```

Comment:
Please also move these up so that this file is alphabetized.

Reply:
Will do.

Reply:
Fixed.